plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -547,7 +553,7 @@ private BaseTable loadTable(String tableName)
 
     private List<String> getActiveFiles(String tableName)
     {
-        return computeActual(format("SELECT file_path FROM \"%s$files\"", tableName)).getOnlyColumn()
+        return computeActual(format("SELECT file_path FROM \"%s$files\" WHERE content = " + FileContent.DATA.id(), tableName)).getOnlyColumn()
```

Comment:
Let's avoid string concatenation and format used together.

Reply:
Just updated the PR.